backend/functions/strava_api_service.py

```
@@ -222,17 +222,18 @@ def collect_pb_effort_activities(self, activity_data: list, access_token: Option
         return pd.DataFrame(pb_effort_data)
 
     def collect_activity_stream_data(
-            self,
-            activity_data: list,
-            vars: Variables,
-            access_token: Optional[str] = None
+        self,
+        activity_data: list,
+        vars: Variables,
```

Ruff (0.15.6)

[error] Function argument `vars` is shadowing a Python builtin

(A002)